tests/Units/GroupEscalationTest.php

```diff
@@ -812,65 +700,85 @@ public function testHistory()
      */
     public function testStandardGroupNotification()
     {
-        global $CFG_GLPI;
+        global $CFG_GLPI, $DB;
 
-        $this->login();
+        $this->initConfig([
+            'use_assign_user_group'              => 1,
+            'use_assign_user_group_creation'     => 0,
+            'use_assign_user_group_modification' => 1,
+            'remove_tech'                        => 1,
+            'remove_group'                       => 1,
+            'show_history' => 1,
```

Comment:
When they are booleans, it is clearer to use `true`/`false` than `1`/`0`.
```suggestion
            'use_assign_user_group'              => true,
            'use_assign_user_group_creation'     => false,
            'use_assign_user_group_modification' => true,
            'remove_tech'                        => true,
            'remove_group'                       => true,
            'show_history'                       => true,
```